src/index/service.js

```diff
@@ -61,3 +63,10 @@ const excerptToHtml
 const formatError
   = err =>
     '<p>Unable to fetch blog posts at this time.</p><!-- ' + err + '-->'
+
+const transformTeam 
+  = (teamMembersObj) => {
+    const id = "owl-team"
+    const styles = "col-md-12"
```

Comment:
After I scanned further down the commit, I can see why you're doing this.  In general, we should keep CSS and HTML out of the JavaScript.  Is there any way to move the element that receives these classes to a page template, rather than a component template?